ads/_config.js

```diff
@@ -899,6 +899,13 @@ export const adConfig = {
     renderStartImplemented: true,
   },
 
+  'videointelligence': {
+    preconnect: [
+      'https://s.vi-serve.com',
+    ],
+    renderStartImplemented: true,
```

Comment:
I noticed you indicate that you support the [renderStart API](https://github.com/ampproject/amphtml/blob/master/ads/README.md#available-apis). However I couldn't receive the `render-start` message when testing with your example. Could you double check your implementation? 
We highly recommend supporting the API for better user experience if you haven't done so.

Reply:
We checked our implementation and we were indeed not calling window.context.renderStart() 
This should be fixed now.
Thank you.